app/src/main/java/org/mifos/mobile/ui/fragments/ReviewLoanApplicationFragment.kt

```diff
@@ -93,37 +95,43 @@ class ReviewLoanApplicationFragment : BaseFragment() {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-        tv_loan_product.text = viewModel.getLoanProduct()
-        tv_loan_purpose.text = viewModel.getLoanPurpose()
-        tv_principal_amount.text = viewModel.getPrincipal().toString()
-        tv_expected_disbursement_date.text = viewModel.getDisbursementDate()
-        tv_submission_date.text = viewModel.getSubmissionDate()
-        tv_currency.text = viewModel.getCurrency()
-        tv_new_loan_application.text = viewModel.getLoanName()
-        tv_account_number.text = viewModel.getAccountNo()
-
-        btn_loan_submit.setOnClickListener {
+        _binding = FragmentReviewLoanApplicationBinding.bind(view)
+        setUpViews()
+
+    }
+
+    private fun setUpViews(): Unit = with(binding) {
```

Comment:
with() returns the the result from the lambda which is the last line, it may be Unit if the last line returns Unit or some other data type too. 
`inline fun <T, R> with(receiver: T, block: T.() -> R): R`
Also using = in place of { in this case makes the code less nested and look cleaner and more readable

Reply:
It is a good practice that you are using `with()` but if the function doesn't suppose to return something then it is better to keep it as default. `=` looks cleaner and readable if a function returns something, not for Unit/Void :)

Reply:
If theres no issue with the extra nesting, then ill make it `{`